week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,7 +35,8 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  let removed = arr.splice(index, 1); // complete this statement
+  return arr;
 }
 
```

Comment:
The function must:
- NOT change the original array - *this hasn't been achieved*

**Hint:** There is another method you can use. Try using `.slice()` array method instead, as `.splice()`, as `splice` changes the original array